docs/site/home.html

```diff
@@ -15,8 +15,11 @@ <h1 class="logo"><img src="site/img/logo-full.svg" alt="Google Cloud Platform" /
       </div>
 
       <div class="quote-box--supplementary">
-        <h2>One-line npm install</h2>
-        <pre>$ npm install --save gcloud</pre>
+        <pre><code class="subtle--blue">$</code> npm install <code class="subtle--blue">--save</code> <strong>gcloud</strong></pre>
+        <h4 class="latest-release subtle" ng-if="latestRelease">
+          Latest Release <strong class="white">{{latestRelease}}</strong>
```

Comment:
Can we wrap the version in a link to either `npm` page for our package or the specific changelog docs for the release on GitHub?